maestro-ios/src/main/java/ios/simctl/SimctlIOSDevice.kt

```diff
@@ -98,7 +98,10 @@ class SimctlIOSDevice(
     }
 
     override fun setLocation(latitude: Double, longitude: Double): Result<Unit, Throwable> {
-        TODO("Not yet implemented")
+        runCatching {
+            Simctl.setLocation(deviceId, latitude, longitude)
+        }
+        return Ok(Unit)
```

Comment:
We should actually return result of runCatching:

```
runCatching {
            Simctl.setLocation(deviceId, latitude, longitude)
}
```

The result of above is either Ok or Err that helps caller telling if the call actually succeeded or failed